app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -70,6 +73,17 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     }
   };
 
+  var getVerbosityTypes = function() {
+    vm['verbosity_types'] = {
```

Comment:
@syncrou currently Angular forms are being populated using API calls, but we don't have back-end or API support to get a list of Verbosity types at the moment so i have hard coded this list for now.